### PR TITLE
PIM-3071: add cookbook for date localizer

### PR DESCRIPTION
**Pull request by skeleton**

### CODE REVIEW

**Comment by solivier** [comment]
GTM

File: cookbook/localization/index.rst [hunk 1]
```diff
@@ -41,3 +42,22 @@ If you want to add your own separator, you have to configure it in your `app/con
             - { value: '.', label: 'dot (.)' }
             - { value: ',', label: 'comma (,)' }
             - { value: '⎖', label: 'apostrophe (⎖)' }
+
+How to Add a Date Format in Import
+----------------------------------
+
+Some basics formats have been added in configuration.
```
Comment (pierallard) [inline cookbook/localization/index.rst]: basic*

File: cookbook/localization/index.rst [hunk 2]
```diff
@@ -41,3 +42,22 @@ If you want to add your own separator, you have to configure it in your `app/con
...
+If you want to add your own format, you have to configure it in your `app/config/config.yml` file.
+
+.. code-block:: yaml
+
+    # /app/config/config.yml
+    pim_localization:
+        format_dates:
```
Comment (sumbobyboys) [inline cookbook/localization/index.rst]: date_formats

**Comment by sumbobyboys** [comment]
GTM after comment fixes

**Pull request merged**